assets/js/monero-gateway-order-page.js

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright (c) 2018, Ryo Currency Project
+*/
+function monero_showNotification(message, type='success') {
+    var toast = jQuery('<div class="' + type + '"><span>' + message + '</span></div>');
+    jQuery('#monero_toast').append(toast);
+    toast.animate({ "right": "12px" }, "fast");
+    setInterval(function() {
+        toast.animate({ "right": "-400px" }, "fast", function() {
```

Comment:
Right: -400px?

Reply:
Correct, negative values are valid for top, left, bottom, right and are used to hide something off screen.

Reply:
Learn CSS standards. To hide something on screen: `display: none`.

Reply:
I'm sorry. My intent was not to attack you on personal. However I appreciate your work.